code/player/camera/ThirdPersonSpectateCamera.cs

```diff
@@ -36,17 +37,7 @@ public override void Update()
 
             if (!player.IsSpectatingPlayer || Input.Pressed(InputButton.Attack1))
             {
-                List<TTTPlayer> players = Utils.GetAlivePlayers();
-
-                if (players.Count > 0)
-                {
-                    if (++_targetIdx >= players.Count)
-                    {
-                        _targetIdx = 0;
-                    }
-
-                    player.CurrentPlayer = players[_targetIdx];
-                }
+                player.UpdateObservatedPlayer();
```

Comment:
Should we have a param into `UpdateObservatedPlayer` that either decrements or increments? Goes forwards and backwards? Or we can leave this in for another PR as well.

Reply:
I'd just add this if it's needed at some point or requested :)